gitolize.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+# A script wrapping a shell command so that it runs with a directory populated from a git repository.
+# In the write mode, the repository is locked with a temporary file and
```

Comment:
This should probably be updated, as we use `locks*` branches for locking rather than a temporary file in the main branch.

Reply:
Done in 6b9ec86ad17127c3c33d41829bd9432b9eea40ae .